doc/syntax/operators.rdoc

```diff
@@ -0,0 +1,75 @@
+= Operators
+
+In Ruby, operators such as <code>+</code>, are defined as methods on the class.
+Literals[rdoc-ref:syntax/literals.rdoc] define their methods within the lower
+level, C language. String class, for example.
+
+Ruby objects can define or overload their own implementation for most operators.
+
+Here is an example:
+
+  class Foo < String
+    def +(str)
+      self.concat(str).concat("another string")
+    end
+  end
+
+  foobar = Foo.new("test ")
+  puts foobar + "baz "
+
+This prints:
+
+  test baz another string
+
+What operators are available is dependent on the implementing class.
+
+== Operator Behavior
+
+How a class behaves to a given operator is specific to that class, since
+operators are method implementations.
+
+When using an operator, it's the expression on the left-hand side of the
+operation that specifies the behavior.
+
+  'a' * 3         #=> "aaa"
+  3 * 'a'         # TypeError: String can't be coerced into Integer
+
+== Logical Operators
+
+Logical operators are not methods on the Ruby Object, and therefor cannot be
```

Comment:
Do you intend this `Object` as `::Object` class?

Reply:
This would also be true of `BasicObject`.  I recommend removing the ` on the Ruby Object` part.

Reply:
Pushed up a commit.